_pages/oidc.md

```diff
@@ -436,7 +436,7 @@ To log out a user, send them to the `/openid_connect/logout` endpoint with the f
 - **post_logout_redirect_uri**
   The URI Login.gov will redirect to after logout. **This must also be registered with the Login.gov IdP in advance.**
 
-- **state**
+- **state** (optional)
   A unique value at least 22 characters in length used for maintaining state between the request and the callback. This value will be returned to the client on a successful logout.
```

Comment:
Is it worth spelling out how this is used a BIT while we are here?   Maybe "This value will be returned to the client on a successful logout as a parameter of `state` added to the `post_logout_redirect_uri` making it visible to the SP."

Reply:
sure! more clarity is always better 😄